src/CoreGraphics/CGColorSpace.cs

```diff
@@ -395,20 +403,52 @@ public byte[] GetColorTable ()
 		}
 			
 		[DllImport (Constants.CoreGraphicsLibrary)]
+#if !NET
 		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'CreateIDCCData' instead.")]
 		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use 'CreateIDCCData' instead.")]
 		[Deprecated (PlatformName.TvOS, 11, 0, message: "Use 'CreateIDCCData' instead.")]
 		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use 'CreateIDCCData' instead.")]
+#else
+		[UnsupportedOSPlatform ("ios11.0")]
+		[UnsupportedOSPlatform ("tvos11.0")]
+		[UnsupportedOSPlatform ("macos10.13")]
```

Comment:
below 10.14 min

Reply:
Some of these are for Unsupported though. Should I remove these for the private ones maybe or do all of these need to be removed?